lib/handlers/formatting.js

```diff
@@ -29,7 +29,8 @@ import { getFormatterTypeByKey } from '../formatters';
 import { logger } from '../logger';
 
 export class FormattingHandler {
-    constructor(ceProps) {
+    constructor(ceProps, maybeEnqueue) {
+        this.enqueue = maybeEnqueue || (async job => await job());
```

Comment:
Is there a case where we wouldn't want to enqueue the formatting?

Reply:
no - but there are some cases where we don't have a queue handy. We discussed some other possible fixes to this whole issue in the channel, so I'm going to "draft" this PR for now and look at those.